files/goerli/deployedContracts.json

```diff
@@ -0,0 +1,29 @@
+{
+    "l1UpgradeExecutorLogic": "0x625cc9ea502fE5B937984D8DD81aD39ef6622D51",
```

Comment:
will this be the canonic point of reference going forward for contract addresses?

Reply:
that's what I'm thinking